tasks/rtloader.py

```diff
@@ -37,6 +37,9 @@ def build(ctx, install_prefix=None, python_runtimes=None, cmake_options='', arch
     here = os.path.abspath(os.path.dirname(__file__))
     dev_path = os.path.join(here, '..', 'dev')
 
+    if cmake_options.find("-G") == -1:
```

Comment:
Is there any chance `Unix Makefiles` is not the default in some no Windows env?

Reply:
FWIW, the `appveyor` script uses `msys makefiles`; I'm not sure what the difference is (or why that would be).